app/controllers/idv/phone_errors_controller.rb

```diff
@@ -10,7 +10,7 @@ class PhoneErrorsController < ApplicationController
     before_action :ignore_form_step_wait_requests
 
     def warning
-      @remaining_attempts = rate_limiter.remaining_count
+      @remaining_submit_attempts = rate_limiter.remaining_count
```

Comment:
I suspect this attribut/property is used in ui erb file? If we renamed it, we will need rename there if used.

Reply:
Yes, that's correct! I renamed it [here](https://github.com/18F/identity-idp/pull/10011/files#diff-d867346e7bf15348c44f26695694d6ebdd97bd38ebc9d7d93199ebefd6a3baa2L31).

[The var we talked about above in the timeout method](https://github.com/18F/identity-idp/pull/10011#discussion_r1473575190), I can't find in use anywhere outside the spec. It should be in use by `app/views/idv/phone_errors/timeout.html.erb`, but [it isn't](https://github.com/18F/identity-idp/blob/main/app/views/idv/phone_errors/timeout.html.erb), even in `main`.

That's why I think maybe that one could be deleted entirely? But it does seem outside the scope of this PR.

Reply:
@night-jellyfish , how about here?
https://github.com/18F/identity-idp/blob/eb7aa033a183f3792ff3031e933626197869a7b5/app/views/idv/phone_errors/warning.html.erb#L30-L32

Reply:
Yep! That's the one I changed in the first link in my previous comment.